app/src/main/res/values/strings.xml

```diff
@@ -58,4 +58,5 @@
     <string name="i_understand">I Understand</string>
     <string name="websocket_no_network">Waiting for network</string>
     <string name="connection">%s@%s</string>
+    <string name="no_messages_yet">There are no messages, yet.\nPull to refresh or send a message.</string>
```

Comment:
I think we can omit the `Pull to refresh` because the view should update automatically on new messages.

Reply:
What about "Send a message to Gotify and it will appear here."?

Reply:
Yes, seems good.